go.mod

```diff
@@ -254,6 +254,7 @@ replace (
 	github.com/go-kit/log => github.com/go-kit/log v0.1.0
 	github.com/go-logfmt/logfmt => github.com/go-logfmt/logfmt v0.5.0
 	github.com/go-logr/logr => github.com/go-logr/logr v0.4.0
+	github.com/go-martini/martini => github.com/go-martini/martini v0.0.0-20170121215854-22fa46961aab
```

Comment:
we don't depend on this at HEAD today, and this looks unmaintained... what's pulling this in?